string my_string
float32 my_float
